### PR TITLE
feat: remove the total filed from query conditions

### PR DESCRIPTION
**Pull request by Fine0830**
Remove the total filed from query conditions.

Video

https://user-images.githubusercontent.com/20871783/169232937-24f32361-d0d3-437b-a47e-3f2b6d3bfc0d.mov

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/views/dashboard/related/trace/Filter.vue [hunk 1]
```diff
@@ -100,8 +110,8 @@ const selectorStore = useSelectorStore();
 const dashboardStore = useDashboardStore();
 const traceStore = useTraceStore();
 const traceId = ref<string>("");
-const minTraceDuration = ref<string>("");
-const maxTraceDuration = ref<string>("");
+const minTraceDuration = ref<number>(NaN);
+const maxTraceDuration = ref<number>(NaN);
```
Comment (wu-sheng) [inline src/views/dashboard/related/trace/Filter.vue:114]: Is this for the duration query bug fix? If so, update the change log when sync to the main repo.

File: src/views/dashboard/related/trace/TraceList.vue [hunk 2]
```diff
@@ -83,8 +83,12 @@ const { t } = useI18n();
...
 const loading = ref<boolean>(false);
 const selectedKey = ref<string>("");
-const pageSize = ref<number>(15);
-const pageCount = ref<number>(5);
+const pageSize = ref<number>(20);
+const total = computed(() =>
+  traceStore.traceList.length === pageSize.value
+    ? pageSize.value * traceStore.conditions.paging.pageNum + 1
+    : pageSize.value * traceStore.conditions.paging.pageNum
+);
```
Comment (wu-sheng) [inline src/views/dashboard/related/trace/TraceList.vue:91]: When total == current page, there would be `no next page number` to continue query, right?

File: src/views/dashboard/related/trace/Filter.vue (hunk 1 above)
Comment (Fine0830) [reply src/views/dashboard/related/trace/Filter.vue:114]: Yes. I will do.

File: src/views/dashboard/related/trace/TraceList.vue (hunk 2 above)
Comment (Fine0830) [reply src/views/dashboard/related/trace/TraceList.vue:91]: Right...

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
Please sync to the main repo.

**Pull request merged**